examples/datasets/plot_iris_dataset.py

```diff
@@ -35,6 +38,10 @@
 x_min, x_max = X[:, 0].min() - 0.5, X[:, 0].max() + 0.5
 y_min, y_max = X[:, 1].min() - 0.5, X[:, 1].max() + 0.5
```

Comment:
Could you move the import of the PCA as well to the last cell.

Remove these two lines. We can let matplotlib manage those.